.github/workflows/pr_bot.yml

```diff
@@ -32,8 +32,8 @@ jobs:
             var msg = `Thanks for opening a PR! :100:
 
             [A couple initial guidelines](https://github.com/magma/magma/wiki/Contributing-Code#commit-and-pull-request-guidelines)
-            - All commits must be signed off. This is [enforced by \`PR DCO check\`](https://github.com/magma/magma/blob/master/.github/workflows/dco-check.yml).
-            - All PR titles must follow the semantic commits format. This is [enforced by \`Semantic PR\`](https://github.com/magma/magma/blob/master/.github/workflows/semantic-pr.yml).
+            - All commits must be signed off. This is enforced by [\`PR DCO check\`](https://github.com/magma/magma/blob/master/.github/workflows/dco-check.yml).
+            - All PR titles must follow the semantic commits format. This is enforced by [\`PR Check Title Or Commit Message\`](https://github.com/magma/magma/blob/master/.github/workflows/semantic-pr.yml).
```

Comment:
Please drop the commit `478ff46f3ef445c8d8129b09be60b6e671a40130` and rebase on magma/master - there is some mix-up with the commits here.

Reply:
Yeah, just noticed the same. That's odd...

Reply:
OK, it's gone.